packages/buildessential.rb

```diff
@@ -3,12 +3,19 @@
 class Buildessential < Package
   description 'A collection of tools essential to compile and build software.'
   homepage ''
-  version '1.0'
+  version '1.0-1'
 
   is_fake
 
   depends_on 'gcc'
   depends_on 'linuxheaders'
   depends_on 'make'
   depends_on 'pkgconfig'
+  
+  # typically required library to compile source code using "./autogen.sh"
+  depends_on 'automake' 
+  depends_on 'libtool'  
+  depends_on 'intltool' 
+  depends_on 'patch'
```

Comment:
I'm going to make a bold proposal and suggest we add these also since we have so many xorg packages and since these seem to common build dependencies on several packages.
```
depends_on 'diffutils'
depends_on 'bison'
depends_on 'flex'
depends_on 'util_macros'
```
This might not be a complete list, however.  Just a start.

Reply:
I think we need also xorg_proto. Building these xorg packages, I would suggest all the protocols are installed before. It can save depedency checking